Source/WTF/wtf/PlatformHave.h

```diff
@@ -342,7 +342,7 @@
 #endif
 
 #if (PLATFORM(MAC) && __MAC_OS_X_VERSION_MIN_REQUIRED >= 120000) \
-    || (PLATFORM(IOS) || PLATFORM(MACCATALYST) || PLATFORM(VISION))
+    || (PLATFORM(IOS) || PLATFORM(WATCHOS) || PLATFORM(MACCATALYST) || PLATFORM(VISION))
```

Comment:
This is now equivalent to `PLATFORM(COCOA) && !PLATFORM(APPLETV)`. Which is shorter, but also quite suspicious, as it seems unlikely that tvOS lacks this?

Reply:
From the private header, this SPI is indeed marked available as of `macos(12.0), ios(15.0), watchos(8.0), tvos(15.0)`.